sessions/react-effects-and-localstorage/journal-app-localstorage/src/components/Tabs/index.js

```diff
@@ -0,0 +1,7 @@
+import "./Tabs.css";
+
+function Flex({ children }) {
+  return <div className="tabs">{children}</div>;
+}
+
+export default Flex;
```

Comment:
Is there a reason why we export the function here at the end of the file? I was wondering about the naming as well.

Reply:
I fixed the naming to be consistent with the dir name.